plugins/woocommerce/client/admin/client/homescreen/mobile-app-modal/index.tsx

```diff
@@ -43,12 +42,6 @@ export const MobileAppModal = () => {
 		ONBOARDING_STORE_NAME
 	);
 
-	if ( isNewBranding() ) {
-		import( './style-new.scss' );
-	} else {
-		import( './style-old.scss' );
-	}
```

Comment:
Are the styles in `new.scss` now applied by default?

Reply:
Yes, the style that was defined differently in the "old" and "new" files has now been incorporated in `style.scss`, taking the new value.